### PR TITLE
Display namespace when SUBJECT is a ServiceAccount

### PR DESCRIPTION
**Pull request by fjammes**
This PR solves the issue: https://github.com/alcideio/rbac-tool/issues/112

### CODE REVIEW

**Comment by fjammes** [comment]
Hello, could you please provide some feedback about this issue?

**Comment by gadinaor-r7** [comment]
Coming right up

File: cmd/policyrules_cmd.go [hunk 1]
```diff
@@ -110,10 +111,18 @@ rbac-tool policy-rules -o json  | jp "[? @.allowedTo[? (verb=='get' || verb=='*'
 				policies := rbac.NewSubjectPermissionsList(filteredPolicies)
 
 				for _, p := range policies {
+
+					var subject string
+					if p.Subject.Kind == "ServiceAccount" {
+						subject = fmt.Sprintf("%v:%v", p.Subject.Namespace, p.Subject.Name)
```
Comment (gadinaor-r7) [inline cmd/policyrules_cmd.go:117]: I suggest using here '/' instead of ':'

**Review by gadinaor-r7 — CHANGES_REQUESTED** [review]

File: cmd/policyrules_cmd.go (hunk 1 above)
Comment (fjammes) [reply cmd/policyrules_cmd.go]: I have just done this update. Thanks and sorry for the delay

**Review by gadinaor-r7 — APPROVED** [review]
LGTM // Thanks

**Comment by fjammes** [comment]
Hi @gadinaor-r7 , thanks so much for the review. However I do not have permissions to merge the PR, could you please help?

Thanks in advance

**Pull request merged**